.gitignore

```diff
@@ -4,3 +4,4 @@ Cargo.lock
 .idea
 *~
 *.swp
+.DS_Store
```

Comment:
this one omits the `**/` that we added to the driver one, is there any reason why?

Reply:
It totally is 🤦  and that's why we do code review!